src/main/scala/cromwell/engine/package.scala

```diff
@@ -60,7 +60,9 @@ package object engine {
     def getConfigOption(key: String): Option[Config] = if (conf.hasPath(key)) Option(conf.getConfig(key)) else None
 
     val backend = Backend.from(workflowOptions.getOrElse("default_backend", conf.getConfig("backend").getString("backend")))
-    val cacheCalls = getConfigOption("call-caching") map { _.getBooleanOr("enabled", default = false) } getOrElse DEFAULT_CALL_CACHING_VALUE
+    val cacheCalls = workflowOptions.getBoolean("use-cache") getOrElse {
+      getConfigOption("call-caching") map { _.getBooleanOr("enabled", default = false) } getOrElse DEFAULT_CALL_CACHING_VALUE
+    }
```

Comment:
I think this is saying workflow options trumps config options trumps the default?  Maybe it might be more clear with the config option part pulled out to a function?  Also this seems to say that if there's a call caching section with no enabled flag then turn off call caching regardless of the built-in default (which currently happens to be false).